crmsh/upgradeutil.py

```diff
@@ -137,8 +147,7 @@ def upgrade_if_needed():
         return
     nodes = crmsh.utils.list_cluster_nodes(no_reg=True)
     if nodes is not None and len(nodes) > 1 \
-            and _is_upgrade_needed(nodes) \
-            and not crmsh.utils.check_passwordless_between_nodes(nodes):
```

Comment:
Here is the same. `check_passwordless_between_nodes` is useless now, should be removed